bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -2252,46 +2252,58 @@ Add a note to the output namelist about the options given to build-namelist
+    'startup' is an initial run with initial conditions provided.
     'continue' is a restart run.
     'branch' is a restart run in which properties of the output history files may be changed.
 </entry>
 
 <entry id="ssp_rcp" type="char*8" category="default_settings"
-       group="default_settings"  
+       group="default_settings"
        valid_values="hist,SSP1-2.6,SSP3-7.0,SSP5-3.4,SSP2-4.5,SSP1-1.9,SSP4-3.4,SSP4-6.0,SSP5-8.5">
-Shared Socioeconomic Pathway (SSP) and Representative Concentration Pathway (RCP) combination for future scenarios 
+Shared Socioeconomic Pathway (SSP) and Representative Concentration Pathway (RCP) combination for future scenarios
 The form is SSPn-m.m Where n is the SSP number and m.m is RCP radiative forcing at peak or 2100 in W/m^2
 n is just the whole number of the specific SSP scenario. The lower numbers have higher mitigation
 - the higher numbers less mitigation, more than one SSP can result in the same RCP forcing
 hist means do NOT use a future scenario, just use historical data.
-</entry> 
+</entry>
 
 <entry id="mask" type="char*10" category="default_settings"
-       group="default_settings"  
+       group="default_settings"
        valid_values="USGS,gx3v7,gx1v6,gx1v7,navy,test,tx0.1v2,tx0.1v3,tx1v1,T62,cruncep,nldas2">
 Land mask description
-</entry> 
+</entry>
 
 <!-- lnd_tuning_mode, there needs to be a setting for: CRUJRA2024, CRUv7, GSWP3v1, cam6.0, and cam7.0 for each valid physics option; exceptions include clm4_5_CRUJRA2024 and clm6_0_CRUv7 -->
 <entry id="lnd_tuning_mode" type="char*20" category="default_settings"
        group="default_settings"
        valid_values="clm4_5_CRUv7,clm4_5_GSWP3v1,clm4_5_cam7.0,clm4_5_cam6.0,clm4_5_cam5.0,clm4_5_cam4.0,clm5_0_cam7.0,clm5_0_cam6.0,clm5_0_cam5.0,clm5_0_cam4.0,clm5_0_CRUv7,clm5_0_GSWP3v1,clm5_0_CRUJRA2024,clm6_0_GSWP3v1,clm6_0_CRUJRA2024,clm6_0_cam7.0,clm6_0_cam6.0,clm6_0_cam5.0,clm6_0_cam4.0">
 General configuration of model version and atmospheric forcing to tune the model to run under.
-This sets the model to run with constants and initial conditions that were set to run well under 
+This sets the model to run with constants and initial conditions that were set to run well under
 the configuration of model version and atmospheric forcing. To run well constants would need to be changed
 to run with a different type of atmospheric forcing.
 (Some options for the newest physics will be based on previous tuning, and buildnml will let you know about this)
-</entry> 
+</entry>
 
 <entry id="megan" type="integer" category="default_settings"
        group="default_settings"  valid_values="0,1">
 If 1, turn on the MEGAN model for BVOC's (Biogenic Volitile Organic Compounds)
-</entry> 
+</entry>
+
+<entry id="megan_use_gamma_sm" type="logical" category="MEGAN_emissions"
+       group="megan_opts" valid_values="" >
+  If TRUE, use activity factor for soil moisture for MEGAN isoprene emissions.
+<default>Default: FALSE </default>
```

Comment:
Should this be set to true so the PR is b4b, then we can have an answer changing tag that changes the default, as intended for CESM3

Reply:
Actually, I suggest removing the line for Default, since it's not actually used. The default used is in the namelist_defaults_ctsm.xml file. Same should be done above.